convert2rhel/utils.py

```diff
@@ -144,9 +144,10 @@ def run_subprocess(cmd, print_cmd=True, print_output=True):
     )
     output = ""
     for line in iter(process.stdout.readline, b""):
-        output += line.decode()
+        line.decode("utf8")
+        output += line
         if print_output:
-            loggerinst.info(line.decode().rstrip("\n"))
+            loggerinst.info(line.rstrip("\n"))
```

Comment:
Previously the output of `decode()` was added to output and logger. With this we just call decode and then use `line` instead. Does `decode()` modify the string itself?

Reply:
no decode doesn't modify the string itself, that is actually a bug that I introduced thank you for pointing it out!